src/api/inline-api.ts

```diff
@@ -211,13 +213,13 @@ export class DataviewInlineApi {
         });
 
         /** Check for optional CSS. */
-        let cssPath = `${this.app.vault.configDir}/dataviews/${viewName}/view.css`;
-        this.app.vault.adapter.exists(cssPath).then(cssExists => {
-            if (!cssExists) return;
+        let cssPath = `${viewName}/view.css`;
+        let cssFile = this.app.metadataCache.getFirstLinkpathDest( cssPath, sourcePath );
+
+        if ( !cssFile ) return;
 
-            this.app.vault.adapter.read(cssPath).then(viewCSS => {
-                this.container.createEl('style', { text: viewCSS, attr: { scoped: '' } });
-            });
+        this.app.vault.read(cssFile).then(viewCSS => {
```

Comment:
Catch here for same reasons as above.

Reply:
I've added the following `catch` to both `read` calls. Tested throwing errors inside the `then` callback as well as in the `viewFunction`, and they are caught and displayed as expected.

```js
this.app.vault.read( … ).then(data => {
    …
}).catch(error => {
    renderErrorPre(this.container, "Dataview: " + error.stack)
});
```